Dockerfile

```diff
@@ -1,25 +1,28 @@
 #Dockerfile
-FROM ruby:2.3.7
+FROM ruby:2.3.7 AS base
 LABEL maintainer="Scott Bishop - ScottBishop70@gmail.com"
 
-# Install tools & libs
-RUN apt-get update
-RUN apt-get install -y build-essential checkinstall libx11-dev libxext-dev zlib1g-dev libpng-dev libjpeg-dev libfreetype6-dev libxml2-dev nodejs
-
-RUN apt-get install -y imagemagick libmagick++-dev libmagic-dev libmagickwand-dev vim libpq-dev && apt-get clean
+ARG RAILS_MASTER_KEY=""
```

Comment:
Why add the default string?

Reply:
Allows building the docker image without requiring the master key to be passed. It's not used anywhere during build time anyways.